habitat/plan.sh

```diff
@@ -44,6 +44,7 @@ pkg_version() {
 
 do_before() {
   do_default_before
+  hab pkg install core/bundler -c unstable
```

Comment:
you want to include core/bundler in the pkg_build_deps above instead of installing it here. This install is simply going to put it on disk but won't put it on the path and therefore it won't actually get called.

Reply:
@mwrock ok but how do I request the unstable one in pkg-deps?

Reply:
@jaymzh what version corresponds to "unstable" please? We can prolly just build that package and upload it?

Reply:
Beats me, you have access to the repos, not me. Has to be a version enough to support `force_ruby_platform`.

According to @Stromweld unstable is `2.5.19 `. The version we build on is 2.5.22, but as long as bundler can read the file, it'll install the expected version and re-exec, so it doesn't have to be exact. But if you promote the unstable build to the stable channel, it'll be new enough.

Reply:
2.5.19 is in unstable and LTS-2024 so it should get pulled in without a need to explicitly call out `unstable`

Reply:
Ah yeah, that part is working now. Is there a hab package with has shadow.h ? That's missing for the build now.